src/auspex/qubit/qubit_exp.py

```diff
@@ -430,7 +436,7 @@ def init_instruments(self):
             listener.start()
 
         while ready.value < len(self.chan_to_dig):
-            time.sleep(0.3)
+            time.sleep(0.3) #Changed from 0.3
```

Comment:
It seems that there is more committed here than intended, maybe make a separate branch with only the relevant changes?